spot_driver/include/spot_driver/api/default_time_sync_api.hpp

```diff
@@ -36,5 +37,6 @@ class DefaultTimeSyncApi : public TimeSyncApi {
 
  private:
   std::shared_ptr<::bosdyn::client::TimeSyncThread> time_sync_thread_;
+  int8_t timesync_timeout_;
```

Comment:
If you follow @bjin-bdai's [suggestion](https://github.com/bdaiinstitute/spot_ros2/pull/552/files#r1913468357) above, this and other instances could match:

```suggestion
  std::chrono::miliseconds timesync_timeout_;
```